cmd/platform/user.go

```diff
@@ -202,7 +201,7 @@ func userDeactivateCmdF(cmd *cobra.Command, args []string) error {
 	}
 
 	if len(args) < 1 {
-		return errors.New("Enter user(s) to deactivate.")
+		return errors.New("Expected at least one argument. See help text for details.")
```

Comment:
Some of the messages were out of date. I think the arguments had changed since they were originally written. Others weren't clear on exactly what was expected. I opted to change the messages to reference the help doc so that we can avoid duplicating documentation in two places that might drift out of date from one another. I'm open to switching back if we think this is a problem